src/index.ts

```diff
@@ -240,6 +240,8 @@ function barfTurn() {
       myInebriety() <= inebrietyLimit()
     ) {
       useFamiliar($familiar`Space Jellyfish`);
+    } else {
+      setMarginalFamiliar(location, underwater);
```

Comment:
won't underwater never be true for this, given its placement in the else-if chain??

Really, I feel like we only want to do this for barf-turns. I don't want us running a jung man against an embezzler. So it might be worth cutting the underwater parameter entirely.

Reply:
Removed underwater, but retained location because knowing that we are in barf helps us evaluate fairies more accurately (this may be useful when we eventually use the same framework for free fights)